src/enzo/QuantumGetUnits.C

```diff
@@ -0,0 +1,132 @@
+/***********************************************************************
```

Comment:
How are these routines different from the normal `GetUnits` and `CosmologyGetUnits`? I couldn't see any differences.

Reply:
They are the same and have now been removed.